source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRBaseTarget.mm

```diff
@@ -6,8 +6,10 @@
 //
 
 #import "ACRBaseTarget.h"
-#import <FluentUI/FluentUI-Swift.h>
 #import <UIKit/UIKit.h>
+#if defined(ADAPTIVECARDS_APP_EXTENSION)
```

Comment:
`ADAPTIVECARDS_APP_EXTENSION` sounds weird as a flag name. Why not `ADAPTIVECARDS_USE_FLUENT_TOOLTIPS` since that's what it enables?

Reply:
I had a similar name initially. After seeing what others do, I changed to this name. the suffix APP_EXTENSION, used throughout frameworks for this pattern. I didn't include Fluent nor Tooltips in the name because the flag is meant for the subspec, `UIProvider`.

Reply:
@jwoo-msft that doesn't make much sense to me. What about the next feature/capability that a host should be able to disable independently of the FluentUI tooltip? What name will you give that next flag?

Reply:
> @jwoo-msft that doesn't make much sense to me. What about the next feature/capability that a host should be able to disable independently of the FluentUI tooltip? What name will you give that next flag?

that's true...

Reply:
@dclaux, sure, I will change the name. the use case you mentioned is a good one, but it's not a realistic one since we won't be using the preprocessor macro for this purpose anymore once we come up with a general solution for third-party controls.